scripts/count_data_points_after_cleaning.sh

```
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -eo pipefail
+
+tempf=$(mktemp)
+JSON_PATH=$PWD/src/json
+
+for f in $(ls $JSON_PATH/soc_cleaned*); do
```

Shellcheck (0.10.0)

[error] Iterating over ls output is fragile. Use globs.

(SC2045)